Source/BoundaryConditions/PML_RZ.cpp

```diff
@@ -0,0 +1,252 @@
+/* Copyright 2019 Andrew Myers, Aurore Blelly, Axel Huebl
+ * Maxence Thevenet, Remi Lehe, Weiqun Zhang
```

Comment:
Have not yet understood what we are supposed to do with this copyright info in general in our WarpX files, but I guess if we add it, it should probably be as correct as possible (which probably isn't the case in many other existing files). Here I guess there should be only your name, then?
```suggestion
/* Copyright 2021 David Grote
```

Reply:
Yes, I'll correct it. My understanding is that everyone who has made a change to the file is listed.